app/assets/stylesheets/govuk_publishing_components/components/_translation-nav.scss

```diff
@@ -0,0 +1,35 @@
+.gem-c-translation-nav {
+  margin-bottom: $gutter;
+  border-bottom: 1px solid $border-colour;
+  @include responsive-top-margin;
+  @include core-16;
```

Comment:
I know this is just a straight copy from government-frontend, but makes sense to tidy up while we're here - in this case, could use your suggestion of always putting `@includes` at the top of a CSS block

Reply:
Good shout.